### PR TITLE
(CDAP-12262) Fix header of trucate fast action popup

### PR DESCRIPTION
**Pull request by bdmogal**

### CODE REVIEW

File: cdap-ui/app/cdap/components/FastAction/TruncateAction/index.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ import ConfirmationModal from 'components/ConfirmationModal';
 import {Tooltip} from 'reactstrap';
 import T from 'i18n-react';
 import classnames from 'classnames';
+import { getType } from '../../../services/metadata-parser/index';
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/FastAction/TruncateAction/index.js]: this should be 
```
import {getType} from 'services/metadata-parser';
```

we don't use relative paths.

Comment (bdmogal) [reply cdap-ui/app/cdap/components/FastAction/TruncateAction/index.js]: ummm, it seems to work though?

Comment (bdmogal) [reply cdap-ui/app/cdap/components/FastAction/TruncateAction/index.js]: fixed

Comment (ajainarayanan) [reply cdap-ui/app/cdap/components/FastAction/TruncateAction/index.js]: yeah using `jquery` also works but its about consistency. We have aliases defined in webpack for use. If we happen to move this file to another location then you don't have to change all the references. also relative paths are hard to follow. Aliases makes them simpler.

**Review by ajainarayanan — COMMENTED** [review]
👍  Code LGTM
